### PR TITLE
Some visual tweaks

### PR DESCRIPTION
**Pull request by vlad-ghita**
Added visual info to Route header.
Added accesskey on Save button.
Renamed assets to respect naming convention.

Regards,
Vlad

### CODE REVIEW

**Comment by vlad-ghita** [comment]
Here's a screen shot http://postimage.org/image/z14rtu0c3/

**Comment by brendo** [comment]
@designermonkey

**Comment by designermonkey** [comment]
I'm not willing to merge this, as I don't agree with what it does.

**Comment by nils-werner** [comment]
And why exactly?

**Comment by designermonkey** [comment]
Because I think it looks cluttered.

Someone else can do it if they feel it's necessary. I'm not the only collab ;)

**Comment by nils-werner** [comment]
Eh? What's the point of your protest if you genuinely have a problem with it yet it will end up in the codebase anyways?

**Comment by nils-werner** [comment]
AAAHHHHH, SHIT. I wanted to checkout the code on my machine and must have copy-pasted too many lines.

**Comment by nils-werner** [comment]
OK, I've force-reverted `master` and pushed the stuff in `integration` instead.

**Comment by designermonkey** [comment]
I can protest all I want, I think it clutters it and don't think it should be included, but I'm not the entire group consensus.

**Comment by creativedutchmen** [comment]
@vlad-ghita can you post a `pre` and `post` screenshot? I don't really see the difference (yet) :)

**Comment by vlad-ghita** [comment]
@creativedutchmen 

Sure. [Before](http://postimage.org/image/3zw68z9u3/) and [after](http://postimage.org/image/487gpjjo9/). This pull-request simply displays more useful information.

I felt the need to view fast the settings of the routes. Feel free to improve the appearance.

**Comment by nils-werner** [comment]
I like it, makes it much more readable once you have more than just a handful of rules.

**Comment by nils-werner** [comment]
It appears to be missing localization though, right?

**Comment by vlad-ghita** [comment]
The romanian locale? Nevermind it. When someone will scream for it, I'll create one.

**Comment by creativedutchmen** [comment]
Yes, I like it too.

File: content/content.routes.php [hunk 1]
```diff
@@ -27,173 +27,168 @@ public function __viewIndex() {
...
-				$divgroup = new XMLElement('div');
-				$divgroup->setAttribute('class', 'group');
-				$label = Widget::Label();
-				$input = Widget::Input('settings[url-router][routes][][http301]', 'yes', 'checkbox');
-				$label->setValue($input->generate() . ' ' . __('Force re-route even if page exists'));
-				$divgroup->appendChild($label);
-				$divcontent->appendChild($divgroup);
+						$labelfrom = Widget::Label(__('From'));
+						$labelfrom->appendChild(Widget::Input("settings[url-router][routes][][from]", General::sanitize($from)));
+						$labelfrom->appendChild(new XMLElement('p', __('Simplified: <code>page-name/:user/projects/:project</code>'), array('class' => 'help', 'style' => 'margin: 0.5em 0 -0.5em;')));
```
Comment (simoneeconomo) [inline content/content.routes.php:134]: Directly injecting HTML elements into the `__` function is not a good idea. You should replace this and other occurrences as follows:

```
__('Simplified: %s', array('<code>page-name/:user/projects/:project</code>'))
```

**Comment by tonyarnold** [comment]
It's quite hard to read the light grey text on the grey background. Is there a way to pump up the contrast between the text and it's background without ruining the design? 

Also, it would be good to find a way to consistently space the route mappings so that they line up — right now it's a bit of a river of text.

That said, the extra information is useful.

**Pull request merged**